magda-gateway/src/createBaseProxy.ts

```diff
@@ -44,5 +47,28 @@ export default function createBaseProxy(): httpProxy {
         });
     });
 
+    proxy.on("proxyReq", function(proxyReq, req, res) {
+        proxyReq.setHeader("TenantId", "undefined");
+        const host = req.headers.host;
+        let endIndex = host.lastIndexOf(":");
+        if (endIndex < 0) endIndex = host.length;
+
+        let domainName = host.substring(0, endIndex);
```

Comment:
Is this just to cut of the port? I think you can get that from the `URL` object with `.hostname` :)

Reply:
Revised.